modules/prototype-kit/deployment.tf

```diff
@@ -0,0 +1,185 @@
+variable "tags" {
+  type        = map(string)
+  description = "Tags to use for each resource, will have Name appended where applicable"
+}
+
+variable "public_ssh" {
+  type        = string
+  description = "The public SSH key to use for deployment"
+  sensitive   = true
+}
+
+variable "private_ssh" {
+  type        = string
+  description = "The private SSH key to use for deployment"
+  sensitive   = true
+}
+
+resource "aws_key_pair" "prototype_deployment" {
+  key_name   = "mhclg-data-collection-prototype-deployment"
+  public_key = var.public_ssh
+  tags       = var.tags
+}
+
+resource "aws_instance" "prototype_server" {
+  ami                         = "ami-01c835443b86fe988"
+  instance_type               = "t2.micro"
+  key_name                    = aws_key_pair.prototype_deployment.key_name
+  associate_public_ip_address = true
+  security_groups             = [aws_security_group.security_group.id]
+  subnet_id                   = aws_subnet.prototype.id
+
+  root_block_device {
+    volume_size           = 8
+    delete_on_termination = true
+  }
+
+  connection {
+    host        = self.public_ip
+    type        = "ssh"
+    user        = "ec2-user"
+    private_key = var.private_ssh
+  }
+
+  provisioner "local-exec" {
+    command     = "make bundle"
+    working_dir = path.module
+  }
+
+  provisioner "file" {
+    source      = "${path.module}/bundle.tar.gz"
+    destination = "/home/ec2-user/bundle.tar.gz"
+  }
+
+  provisioner "remote-exec" {
+    inline = [
+      "curl --silent --location https://rpm.nodesource.com/setup_14.x | sudo bash -",
```

Comment:
What does this install? Node repos?

Reply:
Yup, I'm assuming the version in the standard repos is still 12?

Reply:
For some reason the standard amazon linux 2 yum repos didn't actually have nodejs.